sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -967,6 +967,138 @@ test_suite(
+                # sw/device/silicon_creator/lib/drivers/alert.h
+                #"OWNER_SW_CFG_ROM_LOCAL_ALERT_CLASSIFICATION": ["0x94949494"] * 16,
+                "OWNER_SW_CFG_ROM_LOCAL_ALERT_CLASSIFICATION": ["0x94949494"] * 16,
+                # Set the alert accumulation thresholds to 0 per class.
+                "OWNER_SW_CFG_ROM_ALERT_ACCUM_THRESH": ["0x00000000"] * 4,
+                # Set the alert timeout cycles to 0 per class.
+                "OWNER_SW_CFG_ROM_ALERT_TIMEOUT_CYCLES": ["0x00000000"] * 4,
+                # Set the alert phase cycles to 0,10,10,0xFFFFFFFF for classes
+                # A and B, and to all zeros for classes C and D.
+                "OWNER_SW_CFG_ROM_ALERT_PHASE_CYCLES": [
+                    "0x0",
+                    "0xa",
+                    "0xa",
+                    "0xFFFFFFFF",
+                    "0x0",
+                    "0xa",
+                    "0xa",
+                    "0xFFFFFFFF",
+                    "0x0",
+                    "0x0",
+                    "0x0",
+                    "0x0",
+                    "0x0",
+                    "0x0",
+                    "0x0",
+                    "0x0",
+                ],
+            },
+        ),
+    ],
+)
+
+otp_alert_digest(
+    name = "shutdown_alert_digest_cfg",
+    otp_img = ":shutdown_alert_owner_sw_cfg",
+)
+
+[otp_image(
+    name = "otp_img_shutdown_alert_{}".format(lc_state.lower()),
+    src = "//hw/ip/otp_ctrl/data:otp_json_{}".format(lc_state.lower()),
+    overlays = STD_OTP_OVERLAYS + [
+        ":shutdown_alert_owner_sw_cfg",
+        ":shutdown_alert_digest_cfg",
+    ],
+) for lc_state in ALERT_LC_STATES]
+
+[bitstream_splice(
+    name = "bitstream_shutdown_alert_{}".format(lc_state.lower()),
+    src = "//hw/bitstream:rom",
+    data = ":otp_img_shutdown_alert_{}".format(lc_state.lower()),
+    meminfo = "//hw/bitstream:otp_mmi",
+    tags = ["vivado"] + maybe_skip_in_ci(lc_state_val),
+    update_usr_access = True,
+) for lc_state, lc_state_val in ALERT_LC_STATES.items()]
+
+[opentitan_flash_binary(
+    name = "rom_e2e_shutdown_alert_config_test_{}".format(lc_state.lower()),
+    srcs = ["rom_e2e_shutdown_alert_config_test.c"],
+    devices = [
+        "fpga_cw310",
+    ],
+    deps = [
+        "//hw/ip/uart/data:uart_regs",
+        "//sw/device/lib/base:abs_mmio",
+        "//sw/device/lib/base:bitfield",
+        "//sw/device/lib/testing/test_framework:ottf_ld_silicon_creator_slot_a",
+        "//sw/device/lib/testing/test_framework:ottf_main",
+        "//sw/device/silicon_creator/lib/drivers:retention_sram",
+        "//sw/device/silicon_creator/lib/drivers:rstmgr",
+    ],
+) for lc_state in ALERT_LC_STATES]
+
+[
+    opentitan_functest(
+        name = "shutdown_alert_{}".format(
+            lc_state.lower(),
+        ),
+        cw310 = cw310_params(
+            bitstream = ":bitstream_shutdown_alert_{}".format(lc_state.lower()),
+            rom_kind = "testrom",
```

Comment:
I think this is a typo. We want to run this with the actual rom, which is already spliced into the bitstream.

```suggestion
```